runner.go

```diff
@@ -12,6 +12,7 @@ type Runner struct {
 	Ctx     context.Context
 	Name    string
```

Comment:
name of.. what? :P

Reply:
Of Task. Probably I will rename it

Reply:
Addressed https://github.com/pellared/taskflow/pull/11/commits/f0644c99a0841567a2b99037df8ac7d398db384a